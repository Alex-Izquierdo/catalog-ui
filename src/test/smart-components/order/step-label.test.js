import React from 'react';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import StepLabel from '../../../smart-components/order/step-label';

describe('<StepLabel />', () => {
  let initialProps;

  beforeEach(() => {
    initialProps = {
      index: 123456789,
      text: 'Foo'
    };
  });

  it('should render correctly', () => {
    const wrapper = mount(<StepLabel { ...initialProps } />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
