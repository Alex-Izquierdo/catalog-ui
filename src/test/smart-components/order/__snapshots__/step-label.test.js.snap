// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StepLabel /> should render correctly 1`] = `
<StepLabel
  index={123456789}
  text="Foo"
>
  <div
    className="requests-step-label"
  >
    <span>
      123456790
    </span>
    <span>
      Foo
    </span>
  </div>
</StepLabel>
`;
